packages/adblocker-benchmarks/Makefile

```diff
@@ -4,6 +4,15 @@ all: deps run
 requests.json:
 	curl https://cdn.cliqz.com/adblocking/requests_top500.json.gz | gunzip > requests.json
 
+# VERSION: e1222d16430d754e8a2499fd8c58f49880d6c283
+./blockers/ublock:
+	git clone https://github.com/gorhill/ublock ./blockers/ublock
+	git clone https://github.com/uBlockOrigin/uAssets.git ./blockers/uAssets
```

Comment:
Probably want `git clone --depth=1 ...` ?

This make me realize I should have the shell script do this so that people don't have to mind uAssets when they want to build.

Reply:
Done.

You could consider using [git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules).